src/extras/BitmapText.js

```diff
@@ -484,16 +486,16 @@ export default class BitmapText extends core.Container
             const charCode = parseInt(letter.getAttribute('id'), 10);
 
             const textureRect = new core.Rectangle(
-                parseInt(letter.getAttribute('x'), 10) + texture.frame.x,
-                parseInt(letter.getAttribute('y'), 10) + texture.frame.y,
-                parseInt(letter.getAttribute('width'), 10),
-                parseInt(letter.getAttribute('height'), 10)
+                parseInt(letter.getAttribute('x'), 10) / res + (texture.frame.x / res),
+                parseInt(letter.getAttribute('y'), 10) / res + (texture.frame.y / res),
```

Comment:
This is still going to cause a lint error now that @bigtimebuddy fixed the linter not running on travis. It should be `(x / res) + (frame.x / res)`. No mixed operators.